### PR TITLE
AFE Integration: Copy changes

### PR DESCRIPTION
**Pull request by islemaster**
Some requested changes to copy on our sign-up form and in surrounding marketing materials.  I'm sure there are more copy changes coming, but these are the ones I know about right now.  Details in comments, below.

## Links

[AFE Integration TODO list](https://docs.google.com/document/d/1Ny0pTrvkBMnOKx0CqoNnWTWhwq-oCn9IBYekSjiJMBc/edit)

## Testing story

Manual testing.  You can review changes on your own machine at `localhost.code.org:3000/afe`.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibilityForm.jsx [hunk 1]
```diff
@@ -226,10 +226,17 @@ export default class AmazonFutureEngineerEligibilityForm extends React.Component
...
+              <ShippingAddressFormGroup
+                handleChange={this.handleChange}
+                checkValidationState={this.checkValidationState}
+              />
+              <div>
+                For the purposes of shipping you your Inspiration Kit, your
+                email and school address may be shared with a certified third
+                party.
+              </div>
+            </div>
```
Comment (islemaster) [inline apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibilityForm.jsx:239]: Adds a shipping address disclosure that appears if the address form is visible.  Here's what it looks like on the form:

image

File: pegasus/sites.v3/code.org/public/afe/index.md.erb [hunk 2]
```diff
@@ -9,11 +9,11 @@ theme: responsive
 
 Computer science is a foundational subject for a 21st century education. Students who study CS [perform better in other subjects, excel at problem solving, and are 17% more likely to attend college](https://medium.com/@codeorg/cs-helps-students-outperform-in-school-college-and-workplace-66dd64a69536). By offering CS to your students, you’re helping create new opportunities for them!
 
-Code.org has partnered with Amazon Future Engineer (AFE) to offer middle and high school teachers at underserved schools a growing number of resources and benefits at no cost!
+Code.org has partnered with Amazon Future Engineer to offer middle and high school teachers at underserved schools a growing number of resources and benefits at no cost!
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/public/afe/index.md.erb:12]: Don't use the "AFE" acronym in marketing copy.

File: pegasus/sites.v3/code.org/public/afe/index.md.erb [hunk 3]
```diff
@@ -9,11 +9,11 @@ theme: responsive
...
 
-**The Amazon Future Engineers program is offered to schools with Title 1 status OR that have student enrollment of more than 40% underrepresented minority students and/or 40% students who qualify for free or reduced lunch.**
+**The Amazon Future Engineer program is offered to schools with Title 1 status OR that have student enrollment of more than 40% underrepresented minority students and/or 40% students who qualify for free or reduced lunch.**
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/public/afe/index.md.erb:14]: Don't pluralize Amazon Future Engineer.

File: pegasus/sites.v3/code.org/public/afe/index.md.erb [hunk 4]
```diff
@@ -9,11 +9,11 @@ theme: responsive
...
 
-Scroll down to learn more about what Code.org offers and the benefits of participating through AFE, or if you’re ready, fill out the short form below to see if your school is eligible!
+Scroll down to learn more about what Code.org and Amazon Future Engineer each offer and the benefits of participating, or if you’re ready, fill out the short form below to see if your school is eligible!
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/public/afe/index.md.erb]: Remove acronym, and be inclusive in description of benefits.

File: pegasus/sites.v3/code.org/public/afe/index.md.erb [hunk 5]
```diff
@@ -118,7 +118,7 @@ To learn more about our Professional Learning Program, [click here](https://code
 ## Additional Amazon Future Engineer benefits
 In addition to the Code.org platform and curricula, AFE provides:
 * Paid CSTA+ level membership. Perks include discounts on 3rd-party curricula, physical computing hardware, and the annual CSTA conference. Visit [csteachers.org](https://csteachers.org/) for more details.
-* Eligibility for AFE’s $25,000 teacher of the year award. [Details are here](https://www.amazonfutureengineer.com/teacher-awards).
+* Eligibility for AFE’s $25,000 teacher of the year award. [Details are here](https://blog.aboutamazon.com/community/amazon-announces-teacher-of-the-year-award-recipients).
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/public/afe/index.md.erb:121]: Updated teacher of the year details link.

File: pegasus/sites.v3/code.org/public/afe/start-codeorg.md.erb [hunk 6]
```diff
@@ -112,4 +112,4 @@ To learn more about our Professional Learning Program, [click here](https://code
 <hr>
 
 <a id="eligibility"></a>
-**To participate in the Amazon Future Engineers program, a school must have Title 1 status OR have student enrollment of more than 40% underrepresented minority students and/or 40% students who qualify for free or reduced lunch.*
+**To participate in the Amazon Future Engineer program, a school must have Title 1 status OR have student enrollment of more than 40% underrepresented minority students and/or 40% students who qualify for free or reduced lunch.*
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/public/afe/start-codeorg.md.erb:115]: Don't pluralize Amazon Future Engineer.

File: pegasus/sites.v3/code.org/public/amazon-future-engineer.md.erb [hunk 7]
```diff
@@ -17,12 +17,12 @@ theme: responsive
...
-# Amazon Future Engineer proudly supports your classroom 
+# Amazon Future Engineer proudly supports your classroom
 
 As a Code.org classroom supported by Amazon Future Engineer, **you’re invited to sign-up to receive free additional benefits** such as posters, a free CSTA+ membership, internship and scholarship opportunities, and access to in-depth computer science teaching and student resources.
- 
-[Amazon Future Engineer (AFE)](https://www.amazonfutureengineer.com/) is a four-part, childhood-to-career program aimed at inspiring and educating students from underrepresented and underserved communities to try computer science and coding. This program provides robust resources tailored to students in grades K-12, and teachers as well. 
- 
+
+[Amazon Future Engineer](https://www.amazonfutureengineer.com/) is a four-part, childhood-to-career program aimed at inspiring and educating students from underrepresented and underserved communities to try computer science and coding. This program provides robust resources tailored to students in grades K-12, and teachers as well.
+
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/public/amazon-future-engineer.md.erb:25]: Don't use the "AFE" acronym in marketing copy.

File: pegasus/sites.v3/code.org/public/afe/index.md.erb (hunk 4 above)
Comment (dju90) [reply pegasus/sites.v3/code.org/public/afe/index.md.erb]: Slight wording tweak to make it more digestible: "Scroll down to learn more about what Code.org and Amazon Future Engineer have to offer and the benefits of participating..."

**Pull request merged**